Stat facts/average scale value

AtrisFlax opened the pull request:
Статистика. Отслеживания. Среднее значение шкалы https://github.com/MindboxDevSchool/ItHappened/issues/35

AtrisFlax committed:
Added MostEventfulDayFact fact

AtrisFlax committed:
Added AveragelyScaleCalculator fact

Romarub commented on ItHappened.UnitTests/StatisticsCalculatorsTests/MostEventfulDayCalculatorTest.cs:
```diff
@@ -0,0 +1,158 @@
+
+        [Test]
+        public void FindInTwoTrackerWithFactMostEventfulDay_CalculateSucess()
+        {
```
Это вроде бы тот же файл из предыдущего пулл реквеста. Замечания к нему как и там.

AtrisFlax replied:
Функция проверка функции применимости происходит в этих тестах 
EventTrackerHasOneEvent_CalculateFailure
EventTrackerHasZeroEvent_CalculateFailure

Romarub reviewed (changes requested):
тесты

AtrisFlax committed:
Fix typo
Renamed class AveragelyScaleCalculator -> AverageScaleCalculator.

Added test EventTrackerHasZeroEvent_CalculateFailure
